vespa-cloud/joins/src/main/java/ai/vespa/example/joins/JoinSearcher.java

```diff
@@ -24,13 +24,15 @@
 
 public class JoinSearcher extends Searcher {
 
+    private static final String ATTRIBUTE_ONLY_SUMMARY_CLASS = "attribute-only";
```

Comment:
Is this in schema?

Reply:
That was added to all schemas but [base.sd](https://github.com/vespa-engine/sample-apps/blob/master/vespa-cloud/joins/src/main/application/schemas/base.sd) in #693